src/sonic-yang-models/yang-models/sonic-types.yang

```diff
@@ -11,6 +11,10 @@ module sonic-types {
      * respective YANG files.
      */
 
+    revision 2021-02-23 {
+        description "Added new sonic types.";
+    }
+
     revision 2019-07-01 {
```

Comment:
Can we make use of standard YANG model e.g ietf-inet-types.yang wherever possible instead of defining SONiC specific type definitions?